package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "legacyframework",
+  "version": "4.0.0",
+  "description": "Legacy is a modern, lightweight CSS framework.",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/joexn/LegacyFramework.git"
+  },
+  "author": "joexn",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/joexn/LegacyFramework/issues"
+  },
+  "homepage": "https://github.com/joexn/LegacyFramework#readme",
+  "devDependencies": {
```

Comment:
If there are dependencies (which there are), they should either be checked in or added to .gitignore.